webapp/src/Controller/Jury/UserController.php

```diff
@@ -130,7 +130,7 @@ public function indexAction(): Response
             }
 
             $userdata['user_roles'] = [
-                'value' => implode(', ', array_map(fn(Role $role) => $role->getDjRole(), $u->getUserRoles()))
+                'value' => implode(', ', array_map(fn (Role $role) => $role->getDjRole(), $u->getUserRoles())),
```

Comment:
I think we will forget the comma most of the time, so this will always be flagged.

Reply:
My phpstorm format adds it and I do like it (for less git conflicts and changes) but if others don’t I’m fine with dropping it

Reply:
I'm fine with adding this (assuming here and in my other comments that this won't trigger CI failures or violation messages in the PRs).

Reply:
On second thought: I see there are some cases where there is a single element in an array (across multiple lines) and I think then, if that is really intentionally a single element and there's no more elements expected to be added to that declaration, then the extra comma is confusing. It seems that writing it in a single line statement, then the trailing comma is removed, but that seems inconsistent to me...

Reply:
Then let's not do this rule either